client/app/resources/i18n/domain/Messages_fr_FR.xml

```diff
@@ -128,6 +128,7 @@
    <translation id="domain_tab_DNS_lock_error" qtlid="211307">Une erreur est survenue lors de la demande de réinitialisation des serveurs DNS.</translation>
    <translation id="domain_tab_DNS_update_success" qtlid="284841">La modification des DNS a bien été prise en compte. Veuillez attendre quelques heures pour la propagation aux autres DNS</translation>
    <translation id="domain_tab_DNS_update_error" qtlid="284854">Une erreur est survenue lors de la modification des DNS.</translation>
+   <translation id="domain_tab_DNS_lock_button_title">Réinitialiser les serveurs DNS : non disponible en cas d'activation de DNSSEC</translation>
```

Comment:
Wouldn't `Vous ne pouvez pas réinitialiser vos serveurs DNS pendant l'activation de DNSSEC` conveive the message better ?